packages/migrate/src/__tests__/fixtures/seed-sqlite-js-extra-args/prisma/seed.js

```diff
@@ -0,0 +1,27 @@
+async function main() {
+  await new Promise((resolve) => setTimeout(resolve, 0))
+  console.log('Hello from seed.js')
+
+  if (!process.argv.includes('--my-custom-arg-from-config=my-value')) {
```

Comment:
Is the usage of `=` here tied to the way the seed script was called? (E.g., `--my-custom-arg-from-cli=my-value`). Similarly to the comment above, what happens here if the seed script is called with `--my-custom-arg-from-cli my-value` (with no `=`)?

Rationale for this question: consumers of Node.js CLIs are used to be able to define values by either separating the parameter name from his value via a space, or by using a `=` symbol. It seems here we only support `=`, which can be fine per se, but should maybe be documented somewhere

Reply:
It works the same though better use an args parser, I didn't want to add any extra dep so `.find()` works well for our test here.
Note:
```
// Using parseArgs would be better, but it's only available in v18.3.0, v16.17.0 and up
// https://nodejs.org/api/util.html#utilparseargsconfig
// import { parseArgs } from 'node:util';
```